app/scripts/controllers/push-platform-notifications/services/services.ts

```diff
@@ -177,6 +180,15 @@ export async function activatePushNotifications(
           return;
         }
 
+        sw.addEventListener('notificationclick', (event) => {
+          event.notification.close();
+          const destination = `chrome-extension://${sw.location.host}/home.html#notifications/${notificationData?.id}`;
```

Comment:
As discussed make sure that push links are for all browsers not chrome extension.

Reply:
The fix will be in another PR